lib/matplotlib/backends/backend_qt5.py

```diff
@@ -254,8 +254,11 @@ def mousePressEvent(self, event):
         # flipy so y=0 is bottom of canvas
         y = self.figure.bbox.height - event.pos().y()
         button = self.buttond.get(event.button())
+        modifiers = "+".join([name for name, mod_key, qt_key in MODIFIER_KEYS
+                if (int(event.modifiers()) & mod_key) == mod_key])
```

Comment:
We have already dropped 2.6 support (but test it on the 1.5.x because we are nice).